config/maintainers.json

```
@@ -130,6 +130,16 @@
       "link_url": null,
       "avatar_url": null,
       "bio": null
+    },
+    {
+      "github_username": "andres-zartab",
+      "alumnus": true,
+      "show_on_website": true,
```

```suggestion
      "show_on_website": false,
```

Set this to false for now. It will later be updated once new maintainers grow into more experienced maintainers.